### PR TITLE
telemetry: Update telemetryGenerator dependency version

### PR DESCRIPTION
**Pull request by willyyhuang**
## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description
Update telemetryGenerator dependency version

## Checklist
- [x] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt [hunk 1]
```diff
@@ -128,7 +128,8 @@ class PluginUpdateManager : Disposable {
                         result = Result.Succeeded,
                         id = "autoUpdateActionRestart",
                         source = SOURCE_AUTO_UPDATE_FINISH_NOTIFY,
-                        component = Component.Filesystem
+                        component = Component.Filesystem,
+                        action = "autoUpdateActionRestart"
```
Comment (rli) [inline plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt:132]: @dhruvigajjar do you know if this is the intent of the action field?

Comment (willyyhuang) [reply plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt:132]: I don't know if this is the intend, but build is failing due to this new required action field post telemetry definition generation hence I have temporarily put the id as action. Let me know if this needs to be changed.

Comment (dhruvigajjar) [reply plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt:132]: This is when user clients chooses an action from notification actions notification provided. 
mostly to track if users dismiss or reauth from these notification.

Comment (dhruvigajjar) [reply plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt:132]: if these default action, we should use `toolkit_showNotication`

File: plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt [hunk 2]
```diff
@@ -139,7 +140,8 @@ class PluginUpdateManager : Disposable {
                         result = Result.Succeeded,
                         id = "autoUpdateActionNotNow",
                         source = SOURCE_AUTO_UPDATE_FINISH_NOTIFY,
-                        component = Component.Filesystem
+                        component = Component.Filesystem,
+                        action = "autoUpdateActionNotNow"
```
Comment (rli) [inline plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt:144]:
```suggestion
                        action = "notNow"
```

File: plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt (hunk 1 above)
Comment (rli) [inline plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt:132]:
```suggestion
                        action = "restart"
```

File: plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt [hunk 3]
```diff
@@ -169,7 +172,8 @@ class PluginUpdateManager : Disposable {
                         result = Result.Succeeded,
                         id = ID_ACTION_AUTO_UPDATE_SETTINGS,
                         source = SOURCE_AUTO_UPDATE_FEATURE_INTRO_NOTIFY,
-                        component = Component.Filesystem
+                        component = Component.Filesystem,
+                        action = ID_ACTION_AUTO_UPDATE_SETTINGS
```
Comment (rli) [inline plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt:176]:
```suggestion
                        action = "showSettingsDialog"
```

File: plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt [hunk 4]
```diff
@@ -149,7 +151,8 @@ class PluginUpdateManager : Disposable {
                         result = Result.Succeeded,
                         id = ID_ACTION_AUTO_UPDATE_SETTINGS,
                         source = SOURCE_AUTO_UPDATE_FINISH_NOTIFY,
-                        component = Component.Filesystem
+                        component = Component.Filesystem,
+                        action = ID_ACTION_AUTO_UPDATE_SETTINGS
```
Comment (rli) [inline plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt:155]:
```suggestion
                        action = "showSettingsDialog"
```

**Review by rli — APPROVED** [review]

**Pull request merged**